src/js/components/award/AwardInfo.jsx

```diff
@@ -0,0 +1,64 @@
+/**
+  * AwardInfoContainer.jsx
+  * Created by Emily Gullo 01/20/2017
+  **/
+
+import React from 'react';
+import moment from 'moment';
+
+import SummaryBar from 'components/award/SummaryBar';
+import AwardInfoBar from 'components/award/AwardInfoBar';
+
+const propTypes = {
+    selectedAward: React.PropTypes.object
+};
+
+export default class AwardInfo extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.getStatus = this.getStatus.bind(this);
+    }
+
+    componentWillReceiveProps(nextProps) {
+        if (nextProps.selectedAward !== this.props.selectedAward) {
+            this.getStatus(nextProps.selectedAward);
+        }
+    }
+
+    getStatus(award) {
```

Comment:
I was attempting to parse it the way it comes back from the API, but you're right, that doesn't jive with the redux state.  I believe it worked some of the time because some of the IDs I was testing with had identical start and end dates.